azerothcore/azerothcore-wotlk
# fix(Core/Spells): Added SPELL_ATTR0_CU_NO_INITIAL_THREAT to some spells.

Fixed #7241



## Changes Proposed:
-  Added `SPELL_ATTR0_CU_NO_INITIAL_THREAT` to some spells.

## Issues Addressed:

- Closes #7241
- Closes https://github.com/chromiecraft/chromiecraft/issues/1337

## Tests Performed:

- Not tested

## How to Test the Changes:

1. Have a pet, or a player, body pull a mob without causing any threat on the mob.
2. Use any of the abilities:
```
Rapid Fire
Aspect of the Hawk
Aspect of the Cheetah
Aspect of the Beast
Aspect of the Pack
Aspect of the Monkey
Track Humanoids
Track Undead
Track Hidden
Track Giants
Track Elementals
Track Demons
Track Beasts
Eyes of the Beast
Eagle Eye
Create Firestone
Healthstone
Soulstone,
Detect Invisibility,
Unending Breath,
Sense Demons
```
Alternatively:
1. Download Omen3 Threat Meter
2. Aggro a mob
3. Use the various abilities to see your threat changing on the Threat Meter.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
